graphql/resolvers/_mutations/updateNewsletterSubscription.js

```diff
@@ -1,18 +1,31 @@
-const { ensureSignedIn } = require('@orbiting/backend-modules-auth')
-const updateNewsletterSubscription = require('../../../lib/mailchimp/updateNewsletterSubscription')
+const { ensureSignedIn, Roles } = require('@orbiting/backend-modules-auth')
 
-module.exports = async (_, args, context) => {
+module.exports = async (_, args, { user: me, t, mail: { updateNewsletterSubscription, errors }, ...context }) => {
```

Comment:
don't do this, we need context to pass it to other resolvers. Take what you need on line 4.